esp/templates/users/userview.html

```diff
@@ -160,7 +160,7 @@
 <br />
 {% for class in classes %}
 {% ifchanged class.parent_program %}{% if not forloop.first %}</ul>{% endif %}{{ class.parent_program.niceName }}<br /><ul>{% endifchanged %}
-{% with show_class_details="True" %}
+{% with show_class_manage_links=True show_class_details=True %}
```

Comment:
Hmm, why bother with the with+conditional here?  It doesn't look like anything else uses the `userview_class_entry.html` template so we could just always show it to keep things simple.